packages/backend/src/database/migrations/20240515102618_remove_table_calculation_filter_prefix.ts

```diff
@@ -0,0 +1,26 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+    const versionsWithTableCalculationFilters = await knex(
+        'saved_queries_versions',
+    )
+        .select('saved_queries_version_uuid')
+        .whereRaw("jsonb_path_exists(filters, '$.tableCalculations')");
+
+    await knex('saved_queries_versions')
```

Comment:
```suggestion
 if (versionsWithTableCalculationFilters.length > 0) {
    await knex('saved_queries_versions')
```

I think this should be added, what do you think? ^

Reply:
yeah, we can add it :)